classlib/src/main/java/org/teavm/classlib/java/lang/StringNativeGenerator.java

```diff
@@ -30,6 +30,24 @@ public void generate(GeneratorContext context, SourceWriter writer, MethodRefere
         if (methodRef.getName().equals("intern")) {
             writer.append("return $rt_intern(").append(context.getParameterName(0)).append(");").softNewLine();
         }
+        if (methodRef.getName().equals("toLowerCaseNative")) {
+            writer.append("return $rt_str($rt_ustr(")
+                    .append(context.getParameterName(1)).append(").toLowerCase());").softNewLine();
+        }
+        if (methodRef.getName().equals("toLocaleLowerCaseNative")) {
+            writer.append("return $rt_str($rt_ustr(")
```

Comment:
$rt_str and $rt_ustr are runtime methods and should be written by 'appendFunction`.